### PR TITLE
Rename to Gnosis Protocol

### PR DESCRIPTION
**Pull request by anxolin**
Just renaming....

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,17 +17,17 @@ pip install -r requirements.txt
 
 ```bash
 # help command
-./dfusion --help
-./dfusion
+./gnop --help
+./gnop
```
Comment (alfetopito) [inline README.md:21]: Yikes.

What about `gprotocol`? `gnoprotcl`? `gnoprotocol`? `gno-cli`? `gp-cli`?

Comment (anxolin) [reply README.md:21]: hehe, I knewww it

what about just `gno`?

Comment (anxolin) [reply README.md:21]: everything but `mesa`!

Comment (alfetopito) [reply README.md:21]: if you want short, `gp`?

Comment (alfetopito) [reply README.md:21]: Because `gno` is for whole Gnosis. And this is not for all of Gnosis, only for Gnosis Protocol. In short, `GP`.

Comment (anxolin) [reply README.md:21]: Ok! I'll use GP, otherwise we can change it anytime in the future too

**Comment by anxolin** [comment]
Ohh @alfetopito ! Maybe  `gp` is not a good name, it's taken. I did `gp` without doing the `./gp` and I pushed my local changes. It's short for `git push` 😂

**Comment by alfetopito** [comment]
> Ohh @alfetopito ! Maybe `gp` is not a good name, it's taken. I did `gp` without doing the `./gp` and I pushed my local changes. It's short for `git push` joy

:(

`gp-cli` ?

**Comment by olegsv** [comment]
gnsp?
gprotocol?
gnprotocol?

**Pull request merged**